java/src/jmri/jmrix/loconet/LnNetworkPortController.java

```diff
@@ -29,6 +29,7 @@ protected LnNetworkPortController(LocoNetSystemConnectionMemo connectionMemo) {
     protected boolean mTranspondingAvailable = false;
 
     protected LnCommandStationType[] commandStationTypes = {
+        LnCommandStationType.COMMAND_STATION_STANDALONE,
```

Comment:
@geowar1, This does _not_ seem like a correct fix, but it is possible that I do not understand the problem.

A "Standalone LocoNet" (mentioned in the initial comment in this P/R) is one which has LocoNet communication features, but does _not_ provide access to any sort of command station.  As such, it does not provide any sort of ability to program DCC decoders or provide any sort of mobile decoder control.

The `LnCommandStationType.STANDALONE_COMMAND_STATION` enum is all about "standalone programmer" features via the command station, or a command station surrogate such as the PR3 in "Standalone Programming" mode or a PR4 in "Standalone Programming" mode.  

That is _not_ the same as "Standalone LocoNet"!

And, if I understand correctly, `LnNetworkPortController` is all about connecting to a LocoNet via a networked configuration, not via a computer-to-USB-to-PR4 connection.

I am confused!

Reply:
There is no STANDALONE_COMMAND_STATION… I assume you mean COMMAND_STATION_STANDALONE.
If you look at the enum for COMMAND_STATION_STANDALONE it's name is "Stand-alone LocoNet"; which almost matches what the documentation says to pick for "Standalone LocoNet" (sans hyphen). I'd already tried all the other options and none of them worked (I couldn't control anything on my layout). As soon as I added this enum to the list and picked it as the station type in my connections preferences I was then able to control my layout via the PR4.
I think that STANDALONE means "no command station".
I'll gladly bow to your expertise; how do I select "Standalone LocoNet" for my command station type in the connections preferences? It's not one of the choices.

Reply:
The problem with adding it this way is that it adds "Stand-alone LocoNet" for interface types which do _not_ support "Stand-alone LocoNet" features, such as the Digitrax PR2.

If your problem is specifically with the PR4, then the solution is _not_ in LnNetworkPortController, but is instead in `jmrix.loconet.pr4.PR4Adapter.java`.  I will issue a separate P/R with the appropriate code, and mention this P/R.

Reply:
Excellent! I'll close this one then… Thanks for the review (and the (correct) fix). ;-)